cosmos/lib/cosmos/models/gem_model.rb

```diff
@@ -84,7 +84,7 @@ def self.install(name_or_path)
         rubygems_url = get_setting('rubygems_url')
         Gem.sources = [rubygems_url] if rubygems_url
         Gem.done_installing_hooks.clear
-        Gem.install(gem_file_path, {:build_args => '--no-document'})
+        Gem.install(gem_file_path, "> 0.pre", :build_args => ['--no-document'], :prerelease => true)
```

Comment:
What is this doing? Is this permanent? How does this affect the release?

Reply:
This is allowing prelease gems.  It shouldn't affect the release gems...